### PR TITLE
UI Changes

### PR DESCRIPTION
**Pull request by myothiha09**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @myothiha09 has signed the CLA. Thanks for the pull request!

**Comment by myothiha09** [comment]
Some concerns:

The SelectRoleShareVM's selectedRole could be MutableLiveData. Temporarily leaving it as normal data and updating through a notifier.

There is a potential grammar error when a collaborator is successfully invited but the information about the invitee could not be retrieved. 

The Snackbar is covering Send Invitation button.

The Send Invitation button needs color for two of the following:
 - Color codes for Pre Lollipop to simulate "Ripple"
 - Color code for disabled state

The Add personal message text button needs color code for the "ripple" effect.

The Snackbar is covering the Send Invitation button (and it appears infinitely long).

File: box-share-sdk/src/main/java/com/box/androidsdk/share/legacy/fragments/DatePickerFragment.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import android.os.Bundle;
+import android.widget.DatePicker;
+
+import com.box.androidsdk.share.R;
+
+import java.util.Calendar;
+import java.util.Date;
+
+
+public class DatePickerFragment extends PositiveNegativeDialogFragment
```
[the remark this answers is not in the document]
Comment (myothiha09) [reply box-share-sdk/src/main/java/com/box/androidsdk/share/legacy/fragments/DatePickerFragment.java]: We will be having two versions of the UI: the Old UI and the new UI. We might wanna use A/B testing in the future so having two versions of the code separated into different folders is useful for organization purposes.

**Comment by myothiha09** [comment]
Under the folder usx, only the following files are changed. Others are just a clone of their legacy counterpart.

InviteCollaboratorsFragment
BoxFragment
CollaboratorsRolesFragment

InviteeAdapter

BoxActivity
BoxInviteCollaboratorActivity

ChipCollaborationView

File: box-share-sdk/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -3,18 +3,34 @@
     <color name="box_sharesdk_primary_text">#de000000</color>
     <color name="box_sharesdk_hint">#80000000</color>
     <color name="box_sharesdk_accent">#00adef</color>
-    <color name="box_sharesdk_background">#ffffffff</color>
+    <color name="box_sharesdk_background">#FFFFFF</color>
```
Comment (doncung) [inline box-share-sdk/src/main/res/values/colors.xml]: keep consistent.

File: box-share-sdk/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -73,20 +78,21 @@
...
     <string name="box_sharesdk_another_person" description="Default name shown when a user without a Box account has been invited">Another Person</string>
     <string name="box_sharesdk_no_collaborators" description="Message shown when no collaborators have been invited for the current item">No collaborators for this item.</string>
     <string name="box_sharesdk_has_already_been_invited" description="The string displayed when a person was already invited, %s being the email of the person">%s has already been invited to this item</string>
-    <string name="box_sharesdk_num_has_already_been_invited" description="The string displayed when some number of people have already been invited, %s being number of people">%s people have already been invited to this item</string>
+    <string name="box_sharesdk_num_has_already_been_invited" description="The string displayed when some number of people have already been invited, %s being number of people">%d people have already been invited to this item</string>
```
Comment (doncung) [inline box-share-sdk/src/main/res/values/strings.xml]: This string is not compatible with the legacy logic.

**Pull request merged**